### PR TITLE
fix(routing): prefer use of routerLink over direct manipulation of the history

### PR DESCRIPTION
**Pull request by kensodemann**

### CODE REVIEW

File: src/pages/EditTeaCategory.tsx [hunk 1]
```diff
@@ -57,7 +55,9 @@ const EditTeaCategory: React.FC<EditTeaCategoryProps> = ({ match }) => {
 
       <IonFooter>
         <IonToolbar>
-          <IonButton fill="outline" expand="block" onClick={handleSaveClicked}>Save</IonButton>
+          <IonButton fill='outline' expand='block' onClick={updateCategory} routerLink='/tabs/home'>
```
Comment (ehorodyski-ionic) [inline src/pages/EditTeaCategory.tsx:58]: I don't know if there would be potential for a race-condition to occur here where routing happens before the click event is emitted.

Comment (kensodemann) [reply src/pages/EditTeaCategory.tsx:58]: That is a good point. I should probably roll that one back.

Comment (kensodemann) [reply src/pages/EditTeaCategory.tsx:58]: Also, I don't think I like the `jsxSingleQuote` option now that I see it in actual JSX...

Comment (kensodemann) [reply src/pages/EditTeaCategory.tsx:58]: I am going to see what the Ionic team does in their code...

Comment (ehorodyski-ionic) [reply src/pages/EditTeaCategory.tsx:58]: Nice catch, I don't like it either for markup.

**Pull request merged**